### PR TITLE
Named branch testing jobs

### PR DESCRIPTION
**Pull request by estute**

### CODE REVIEW

**Comment by estute** [comment]
@benpatterson @jzoldak PTAL when you get a chance. I also added in the view dsl to automatically group named releases as they come along

File: platform/jobs/edxPlatformAccessibilityMaster.groovy [hunk 1]
```diff
@@ -86,10 +83,10 @@ secretMap.each { jobConfigs ->
             githubProjectUrl(JENKINS_PUBLIC_GITHUB_BASEURL + jobConfig['url'])
         }
         parameters {
-            stringParam(params.name, params.default, params.description)
+            stringParam('sha1', jobConfig['defaultBranch'], 'Sha1 hash of branch to build. Default branch : master')
```
Comment (jzoldak) [inline platform/jobs/edxPlatformAccessibilityMaster.groovy]: nit, extra space after colon in help text.
s/branch : master/branch: master

File: platform/views/platformViews.groovy [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.DEFAULT_VIEW
+
+List<String> branchList = [ "edx-platform", // Represents all non-release testing
```
Comment (benpatterson) [inline platform/views/platformViews.groovy]: should this be 'master' instead?

**Comment by benpatterson** [comment]
I had a question, but not a blocker. Do the needful. 👍

**Pull request merged**